xvkBase.hpp

```diff
@@ -151,7 +151,7 @@ namespace xvk { namespace Base {
 			#ifndef XVK_USE_QT_VULKAN_LOADER
 			if (vulkanLib) return true;
 			#ifdef _WIN32
-				vulkanLib = LoadLibrary("vulkan-1.dll");
+				vulkanLib = LoadLibrary(L"vulkan-1.dll"); // Fix issue with LPCWSTR compilation
```

Comment:
I am curious about your specific circumstances where this was causing a problem... 
Can you clarify what issues you had ?
If this was just for your own habit of having an `L` in front of any string, I won't accept this change because this specific string and the name of libraries will never contain extended characters that would require LPCWSTR string.

Reply:
Visual Studio 2019 project generate that error... 
```
1>H:\Data\Projects\VK_KHR_ray_tracing\include\xvk\xvkBase.hpp(154,1): error C2664: 'HMODULE LoadLibraryW(LPCWSTR)': cannot convert argument 1 from 'const char [13]' to 'LPCWSTR'
```

Reply:
Okay fair enough, I did not get that issue when I tried on my side with visual studio 2019, but if some people can have an error with specific configurations it is a perfectly good reason to fix it. 

I have just pushed a fix on the master branch.